RPIPs/RPIP-44.md

```diff
@@ -59,6 +59,7 @@ The exit functionality for a megapool is specified as follows:
       - It MUST NOT be possible for accounts other than the node operator to exit more validators than needed to reduce `deficit` below `exit_deficit`
 - The protocol SHALL use the `withdrawn`, `credit`, and `received` balances to pay off `debt` prior to taking action on exits
 - The protocol MAY use unstaking and staked RPL to decrease `debt` by the corresponding amount prior to taking action on exits
+- The protocol MAY use unclaimed rewards to decrease `debt` by the corresponding amount prior to taking action on exits
```

Comment:
Should "unclaimed rewards" also be listed? Maybe needs further descriptions to if so

Reply:
This is covered in the list in RPIP-43, right? You could link to RPIP-43.

Reply:
Agree RPIP-43 already says this and double define is bad